### PR TITLE
{bio}[intel/2018b] MultiQC v1.7 w/ Python 2.7.15 + 3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/d746d24292daf38dd8def3092df491c3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/399aaa770cf996bd8f835ad14495bf07 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2034.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/89066f6495a496db1665c0b79900c829 for a full test report.

File: easybuild/easyconfigs/p/PyYAML/PyYAML-3.13-intel-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['3ef3092145e9b70e3ddd2c7ad59bdd0252a94dfe3949721633e41344de00a6bf']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('libyaml', '0.2.1'),
+]
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PyYAML/PyYAML-3.13-intel-2018b-Python-2.7.15.eb]: What travis said...

Comment (boegel) [reply easybuild/easyconfigs/p/PyYAML/PyYAML-3.13-intel-2018b-Python-2.7.15.eb]: fixed in e58a0c5d1

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2403.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/002bd93f00a2b909b5824bd6a415c401 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**